app/src/ui/changes/changes.tsx

```diff
@@ -130,7 +130,7 @@ export class Changes extends React.Component<IChangesProps, {}> {
   private onHideWhitespaceInDiffChanged = async (
```

Comment:
Maybe this function name should change but I sorta think not 🤔

Reply:
Makes sense to me to leave it since we are in the `changes.tsx` .. `inChanges` would be redundant.